spec/src/main/asciidoc/chapters/api/template.adoc

```diff
@@ -88,3 +90,100 @@ List<SmsNotification> notifications = template.select(SmsNotification.class);
+
+For example, consider the following data model:
+
+[source,java]
+----
+class Person {
+  private Long id;
+  private MailingAddress address;
+}
+
+class MailingAddress {
+  private String zipcode;
+  private String city;
+}
+----
+
+In this scenario, querying for records based on the zip code at the `MailingAddress` class requires accessing the `address` field at `Person` and the `zipcode` property at `MailingAddress`.
+
+[source,java]
+----
+@Inject
+Template template;
+
+List<Book> books = template.select(Person.class)
+        .where("address.zipcode")
+        .eq("402-775")
+        .orderBy("address.city")
+        .asc()
+        .result();
+
+template.delete(Person.class)
+     .where("address.zipcode")
+     .eq("402-775")
+     .execute();
+----
+
+In the above example, the fluent API query navigates through the properties of the `Person` entity to access the `zipcode` property of the `MailingAddress` embedded class. The `where` clause specifies the path to the `zipcode` property using dot notation (`address.zipcode`). The `orderBy` clause similarly specifies the path to the `city` property for sorting the results by city in ascending order.
+
+This query navigation hierarchy enables developers to construct complex queries traverse multiple levels of entity properties, facilitating flexible and precise data retrieval and manipulation in Jakarta NoSQL.
+
+=== TTL (Time-To-Live) Support
+
+TTL (Time-To-Live) is a feature provided by many NoSQL databases that allows developers to set an expiration time for data stored in the database. When data reaches its TTL, it is automatically removed from the database, freeing up resources and ensuring that it remains efficient and clutter-free.
+
+For Java developers, TTL support is essential for managing data lifecycle and optimizing resource usage. It enables developers to implement caching strategies, manage temporary data, and enforce data retention policies effectively.
+
+While TTL support is valuable, not all NoSQL databases provide native support for TTL. In cases where TTL is not supported, attempting to set a TTL on data may result in an `UnsupportedOperationException` being thrown by the Jakarta NoSQL provider.
+
+Additionally, some NoSQL providers may have limitations on the granularity of TTL values, such as supporting only TTL values specified in certain units (e.g., hours) or rounding TTL values to the nearest supported unit. In such cases, attempting to set a TTL value that does not align with the provider's limitations may result in unexpected behavior or zero TTL being applied.
```

Comment:
I assume you meant Time-To-Live is not applied rather than that it has zero time to live in the database, meaning you would never write it.
```suggestion
Additionally, some NoSQL providers may have limitations on the granularity of TTL values, such as supporting only TTL values specified in certain units (e.g., hours) or rounding TTL values to the nearest supported unit. In such cases, attempting to set a TTL value that does not align with the provider's limitations may result in unexpected behavior or no TTL being applied.
```